src/client/dfs/dfs.c

```diff
@@ -936,11 +936,11 @@ open_sb(daos_handle_t coh, bool create, dfs_attr_t *attr, daos_handle_t *oh)
 	d_iov_set(&iods[i].iod_name, MAGIC_NAME, strlen(MAGIC_NAME));
```

Comment:
any reason not to do same thing here? And for the other iod entries?

Reply:
you are too picky.
but yes i'll update :-)